src/socket.c

```diff
@@ -3439,8 +3453,8 @@ static int handle_socket_input(const char *simbuffer, int simlen, const char *en
 #if WIDECHAR
 		    inbound_decode_str(xsock->buffer, incomingposttelnet,
                         incomingFSM, 0);
-		    handle_socket_input_queue_lines(xsock);
 #endif
+		    handle_socket_input_queue_lines(xsock);
```

Comment:
@warmstarter That line seems to have fixing all issues..

And as I understood code I dont see why it should be inside that `#if`

Reply:
nope, this is the code I'm talking about and the interaction seems to be doing something with SSL+UTF8  in rare circumstances. I'm looking into this for a better fix, but right while this introduces new weird behavior in edge cases, I think it's better to leave it in, just be aware.

Reply:
my mistake, this is not causing an issue. However, I've found a new separate issue because of this, but it existed before this code addition.

Reply:
let me guess though, LP-style MUDs do a bunch of up front negotiation on connections, yeah?